bin/tictactoe.rb

```diff
@@ -1,40 +1,152 @@
-#defining the player class object
-class Player
-  attr_accessor :name, :id, :state
-
-  def initialize(name, id, state)
-    @name = name    #player1 or player2
-    @id =id         # 'X' or 'O'
-    @state = state  # win! or lose!
-  end
+puts "Welcome to TIC TAC TOE"
```

Comment:
Split this codefile into 2. One being the User Interaction controller (`displayController.rb`), and the other one being the file that actually runs the game. This is important so that it we can separate the game interface from the game itself.